### DISCUSSION TITLE
Loom style emoji comments

### DISCUSSION THREAD

**Discussion by jasonLaster** [RecordReplay/devtools#4603] [Ideas]
I wonder if loom style emoji comments might be a fun lightweight way to callout interesting times in the timeline

<img width="1177" alt="Screen Shot 2021-11-20 at 10 52 15 AM" src="https://user-images.githubusercontent.com/254562/142737984-48cef5ab-6b8e-47cc-ba5e-75d79357b966.png">

<img width="921" alt="emojis" src="https://user-images.githubusercontent.com/254562/142737977-1498f621-c33b-4123-89f0-7feb3d2898fa.png">

**Comment by jonbell-lot23** [comment]
I like this a lot, and it ties back to some very clear data we saw in our user research. Comments are fine, but sometimes you want more of a "laser pointer" where you can draw attention (a) without literal words (b) using visual queues to prioritise things.

We should put this in the same mental bucket as:

* Adding colour/emojis to the console so certain things jump out
* The ability to add a comment with zero text
* Global comment adding (for example through a keyboard shortcut)

I wouldn't put this high in our priority stack, but I do think it's a great idea and part of a broader "How can we make certain moments pop more" exploration that we will eventually need to address.

**Comment by jonbell-lot23** [comment]
March 15, 2022:

As I go through our discussion board, I'm stuck by how many of our ideas come back to this "help me make certain moments pop" insight. Definitely worth exploring.
